wagtail/test/settings.py

```diff
@@ -3,7 +3,7 @@
 from django.contrib.messages import constants as message_constants
 from django.utils.translation import gettext_lazy as _
 
-DEBUG = False
+DEBUG = True
```

Comment:
Why is this change needed?

Reply:
Oops, it's not, just leftover from testing 😆

Though honestly I'd really like us to change this (in a separate PR). This was added in #3381, but I'm not convinced it gave any speed up at all. From https://docs.djangoproject.com/en/stable/topics/testing/overview/#other-test-conditions:

> Regardless of the value of the [`DEBUG`](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-DEBUG) setting in your configuration file, all Django tests run with [`DEBUG`](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-DEBUG)=False. This is to ensure that the observed output of your code matches what will be seen in a production setting.

Changing this to `True` would allow us to easily run the testapp locally (without having to configure static files) for testing PRs, since we can reuse the code (e.g. things like the ViewSet definitions) instead of having to modify bakerydemo.